deployments/networks/eth-poa.yml

```diff
@@ -1,3 +1,9 @@
+# Compose file for quickly spinning up a local instance of an Ethereum PoA network.
+#
+# Note that this PoA network is only used for testing, so the configuration settings you see here
+# are *not* recommended for a production environment.
```

Comment:
Perhaps, you might even be more explicit, like: "Do *NOT* use those configuration settings for a production environment" or something like that.

Reply:
The reason I say they're not recommended instead of outright saying you shouldn't use them is because some people may want to expose RPC endpoints publicly (for examples, those who are hosting nodes for Polkadot JS Apps)

Reply:
I assume the offensive line is the `--engine-signer` one. Perhaps  be specific about this one?

Reply:
@adoerr I've made the wording stronger, what do you think?